.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,7 +13,7 @@ Contributors guide: https://github.com/electron/electron/blob/main/CONTRIBUTING.
 - [ ] PR description included and stakeholders cc'd
 - [ ] `npm test` passes
 - [ ] tests are [changed or added](https://github.com/electron/electron/blob/main/docs/development/testing.md)
-- [ ] relevant documentation is changed or added
+- [ ] relevant documentation, tutorials and examples are changed or added
```

Comment:
```suggestion
- [ ] relevant documentation, tutorials and templates are changed or added
```

Reply:
I was thinking the website examples e.g 
<img width="357" alt="image" src="https://user-images.githubusercontent.com/33054982/204402740-fff22d78-a32c-46e0-a6d8-d037d790a25f.png">

Reply:
It might be helpful to have a definition of what good documentation looks like, similar to `testing.md` above for testing? We don't have to include that in this PR, but that may get more to the meat of what we expect people to add when we ask for tutorials, examples, etc

Reply:
I don't have strong opinions on leaving the "examples" there, but I do think that "templates" (e.g. quick-start, Forge templates, etc.) should be added especially since it's in the PR title and descriptions.